notificationworkerlambda/ecr.yaml

```diff
@@ -1,10 +1,15 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Description: ECR repositories for MSS notification worker lambdas
+Parameters:
+  RepositoryName:
+    Description: Name of the ECR repository
+    Type: String
+    Default: notificationworker-lambda-images
 Resources:
   NotificationLambdaRepository:
     Type: AWS::ECR::Repository
     Properties:
-      RepositoryName: notificationworker-lambda-images
+      RepositoryName: !Ref RepositoryName
```

Comment:
I imagine this is fine as you've tested, but is it ok to call both the property and the parameter exactly the same?

Reply:
good point, I hadn't even noticed, but as you suggested, it works in testing, so I guess the answer is yes, it is ok :-)